### PR TITLE
How to enable SSL pinning on mobile

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Enable SSL Pinning on the mobile apps when you build your own.

Do not merge until this functionality is released.
Related PR's: 
https://github.com/mattermost/react-native-network-client/pull/129
https://github.com/mattermost/mattermost-mobile/pull/8055

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-59055

### CODE REVIEW

File: site/content/contribute/more-info/mobile/build-your-own/ssl-pinning.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+openssl s_client -connect yourserver.com:443 -showcerts < /dev/null | openssl x509 -outform DER -out yourserver.cer
+```
+
+Alternatively, to save it in PEM format:
+
+```sh
+openssl s_client -connect yourserver.com:443 -showcerts < /dev/null | openssl x509 -outform PEM -out yourserver.crt
+```
+
+#### Naming the Certificate Files
```
Comment (jwilander) [inline site/content/contribute/more-info/mobile/build-your-own/ssl-pinning.md:43]:
```suggestion
#### 2. Naming the Certificate Files
```

**Review by jwilander — APPROVED** [review]
LGTM - just one typo

**Review by coltoneshaw — APPROVED** [review]
Looks good! Only feedback would be really underpinning the warning there of requiring coordination between teams.

**Comment by enahum** [comment]
@cwarnermm any idea how to achieve Colton's suggestion ?

**Comment by cwarnermm** [comment]
@enahum @coltoneshaw - My latest commit adds an Important Note directly after the disadvantages list:
Screenshot 2024-07-09 at 3 45 37 PM

**Review by sbishel — APPROVED** [review]
Nice, now I know what SSL Pinning is...

**Review by esarafianou — APPROVED** [review]

**Pull request merged**